web/templates/components/icon_button.hbs

```diff
@@ -1,3 +1,3 @@
-<button class="{{#if custom_classes}}{{custom_classes}} {{/if}}icon-button {{#if squared}}icon-button-square {{/if}}icon-button-{{intent}}" {{#if data-tooltip-template-id}}data-tooltip-template-id="{{data-tooltip-template-id}}"{{/if}} tabindex="0">
+<button type="button" class="{{#if custom_classes}}{{custom_classes}} {{/if}}icon-button {{#if squared}}icon-button-square {{/if}}icon-button-{{intent}}" {{#if data-tooltip-template-id}}data-tooltip-template-id="{{data-tooltip-template-id}}"{{/if}} tabindex="0">
```

Comment:
Probably rather than these very long lines, we should consider doing some line-wrapping in these components and the handlebars partials that include them, to put one field per line or something.

Reply:
Sure, I will work on this as a follow-up.